lib/tasks/templates_renderer.rake

```diff
@@ -0,0 +1,36 @@
+# TRANSLATORS: do not translate
+desc <<-END_DESC
+Render templates from a directory
+
+Available conditions:
+  * FOLDER        => Path to the folder where the templates are stored
+
+  Example:
+    rake templates:render FOLDER=/tmp/community-templates/provisioning_templates
+
+END_DESC
+
+namespace :templates do
+  task render: :environment do
+    path = ENV['FOLDER'].to_s
```

Comment:
I can live with that but `FOLDER`? Really? ;-) Sounds like a `PATH` to me.

Reply:
Holy crap that was the reason why it could not find `git` on PATH. Huh. Nice.

Reply:
Folder vs Directory, round 1, fight :-) I don't mind, just wanted to be funny.

Reply:
I think I like `DIRECTORY` more. :-)

Reply:
Rake tasks can have arguments, making it self documenting. Would that be a better idea?

Reply:
I chose env variables because they have named arguments. If we extend the task to write the rendered templates to files, we'd need a second argument.

But I must admit this looks pretty nice:
```
foreman-rake templates:render['/tmp/community-templates/provisioning_templates']
```